### ISSUE TITLE
Empty values are not correctly transmitted

### ISSUE THREAD

**Issue by olebole** [django-daiquiri/daiquiri#49]
This issue may or may not related to #48:

When one queries a table column that has empty values, it seems that the invalid value `'none'` is transferred:
```
import pyvo
x_url = 'https://gaia.aip.de/tap'
qry = 'SELECT top 10 priam_flags FROM gdr2.gaia_source'
tap_service = pyvo.dal.TAPService(x_url)
tap_service.run_sync(qry)
```
gives a stack trace that ends in
```
DALFormatError: ValueError: None:20:28: \
   ValueError: invalid literal for int() with base 10: 'none' (in row 1, col 'priam_flags')
```
For me it is a bit unclear what should be transmitted for empty cells, but the string  `'none'` makes it impossible to query the column with PyVO. It seems that an empty string would fit better, see  

https://github.com/astropy/astropy/blob/a0f5940bd00ff7259ed6b2391bfddecd007a2a43/astropy/io/votable/converters.py#L821-L842
```Python
            if value == '':
                if config['version_1_3_or_later']:
                    mask = True
                else:
                    warn_or_raise(W49, W49, (), config, pos)
                if self.null is not None:
                    value = self.null
                else:
                    value = self.default
            elif value == 'nan':
                mask = True
                if self.null is None:
                    warn_or_raise(W31, W31, (), config, pos)
                    value = self.default
                else:
                    value = self.null
            elif value.startswith('0x'):
                value = int(value[2:], 16)
            else:
                value = int(value, 10)
```

I however didn't find where this is actually set in Daiquiri.

**Comment by jochenklar** [comment]
I tested it locally, and it works fine with proper `NULL` values in the database. Maybe the data in `gdr2.gaia_source` is not using NULL but something weird.

**Comment by olebole** [comment]
@galkinAIP @harenk can you check that?

**Comment by jochenklar** [comment]
Sorry, now I can reproduce it (it fails for int columns). I will try to fix it.

**Comment by jochenklar** [comment]
Ok fixed in 7ca6d11302a9534b3f23d69cd12dadffada76173. It was only affecting tap/sync since in this case pg_dump is not involved, so that `NULL` values are not `'NULL'` strings, but Python `None`.
